src/ResourceManager/Advisor/Commands.Advisor/help/Set-AzureRmAdvisorConfiguration.md

```diff
@@ -0,0 +1,218 @@
+---
+external help file: Microsoft.Azure.Commands.Advisor.dll-Help.xml
+Module Name: Az.Advisor
+online version:
+schema: 2.0.0
+---
+
+# Set-AzureRmAdvisorConfiguration
+
+## SYNOPSIS
+Updates or creates the Aure Advisor Configration.
```

Comment:
*Azure

Please be especially sure that there are no spelling mistakes for customer facing documentation. I made some edits, but please check across the board. I'll look more closely next time to make sure there are none remaining.